lib/util/devops.js

```diff
@@ -237,6 +243,178 @@ const DevOps = {
         }
         return delta;
     },
+    /**
+     * wrapper around DevOps.getDeltaList, Builder.buildTemplate and M
+     *
+     * @param {object} properties project config file
+     * @param {string} range git commit range
+     * @param {TYPE.SkipInteraction} [skipInteraction] allows to skip interactive wizard
+     */
+    async buildDeltaDefinitions(properties, range, skipInteraction) {
```

Comment:
part of me feels like this may benefit from being broken down into a couple of diff methods.

Reply:
i agree. not really in scope here though but let me create a ticket.

Reply:
created #318